apps/cmp.c

```diff
@@ -2924,15 +2924,13 @@ int cmp_main(int argc, char **argv)
     ret = 0;
 
     if (opt_batch) {
-#ifndef OPENSSL_NO_ENGINE
         UI_METHOD *ui_fallback_method;
 # ifndef OPENSSL_NO_UI_CONSOLE
```

Comment:
You should fix the precompiler directive indentation here if you're removing the outer `#ifndef`

Reply:
Ah, right - done.